_posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md

```diff
@@ -0,0 +1,102 @@
+category: events
+tags: ["hackathon", "company", "company culture"]
+authors: ["Christina"]
+about_authors: ["cgebken"]
+---
+
+The Hackathon is a recurring entry in our calendars.
+Each year we plan two days filled with interesting topics and lots of fun.
+But what does it really mean to take part in our Hackathon?
+Let me take you on a little time journey through the last month to give you some insights into the preparation and the event itself.
+
+## Three weeks ago
+
+It's getting serious guys.
+We were asked to brainstorm about possible topics and to add them to a Confluence list.
+These questions swirled through our minds:
+- What would help us in our everyday office life?
+- Which topic are on our I-always-wanted-to-work-on-that list?
+- Which topics are great but didn't make it yet and might never make it to one of the teams' sprints?
+
+My team member Birgit and I also brainstormed and added a topic to the list: Improving the [About us page](/about/) of the DevPortal.
+
+## Two weeks ago
+
+All topics were set and added to the list.
+Now, it was time to decide which topic we'd like to join.
+Let me tell you, it was a tough decision.
+So many interesting but different topics.
+Some of them aimed to improve our processes and workflows, such as a standardized on-boarding process, unified PR pipelines, a performance test, or a Circle CI wallboard.
+Others had a closer look at our products and merchants, e.g. a UX check for integrating apps, an app for importing products via CSV, or an instore checkout.
+
+These questions helped me to decide which topic I'd like to join:
+- How can I support the team that is working on that topic?
+- What will I learn when participating?
+- Which team members will I have? 
+
+Especially the last point was one of the most important ones for me.
+A Hackathon is the perfect chance to work together with colleagues that I don't work with on a daily basis.
+Maybe even with colleagues from different locations.
+This way, we can even build little "feature teams" for the two days.
+
+As Birgit covered the About us page, I took the chance to join a group which had the goal to categorize our merchants and their shops.
+The results of such a categorization could help Product Management to make decisions on future features or improvements.
+And of course, I also had my personal goal: Get some improvement ideas for our help articles.
+
+## Last Wednesday
+
+The time has come.
+It's Hackathon!
+And what could be the best way to start the first day?
+Right - Breakfast! 
+
+{% twitter https://twitter.com/epagesdevs/status/1110812533349322752 %}
+
+The room was packed with ePagees.
+Our colleagues from Jena, Barcelona, and Bilbao already arrived the day before and everyone was chatting... and eating.
+Around 10 am the last buns and bagels disappeared, and our CTO officially kicked off the event.
+
+During the rest of the day, everyone was diving into their topics, brainstorming, discussing, hacking, and prototyping.
```

Comment:
For the rest of the day, each team was diving into its topic, brainstorming, discussing, hacking, or developing prototypes.